ehr-proxy/utils.py

```
@@ -0,0 +1,33 @@
+    """
+    if height_m <= 0:
+        raise ValueError("Height must be positive")
+    return weight_kg / (height_m ** 2)
+
+
+def fahrenheit_to_celsius(temp_f: float) -> float:
+    """Convert Fahrenheit to Celsius."""
+    return (temp_f - 32) * 5/9
+
+
+def calculate_egfr(creatinine: float, age: int, is_female: bool) -> float:
```

Ruff (0.14.10)

Boolean-typed positional argument in function definition

(FBT001)